src/color.css

```diff
@@ -0,0 +1,11 @@
+:root {
+  --color-one: #fff7fc;
+  --color-two: #691643;
+  --color-three: #a26769;
+  --color-four: #e0c2c0;
+  --color-five: #57394a;
+  --color-six: #57394a;
+  --color-seven: #ffffff;
+  --color-eight: #9a9a9a;
+  --color-nine: #2e1e27;
+}
```

Comment:
yes i would definately speccify the colours for each e.g. --color-red etc.. that might save you a stroke later..